components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -1197,4 +1195,39 @@ public static String validateTenantDomain(String tenantDomain) throws UserStoreE
         return null;
     }
 
+    /**
+     * build the error response
+     * @param status
+     * @param message
+     * @return decoded response
+     * @throws org.wso2.carbon.identity.base.IdentityException
+     */
+    public static String buildErrorResponse(String status, String message)
+            throws IdentityException, IOException {
+        ErrorResponseBuilder respBuilder = new ErrorResponseBuilder();
+        List<String> statusCodeList = new ArrayList<String>();
+        statusCodeList.add(status);
+        Response response = respBuilder.buildResponse(null, statusCodeList, message);
+        String resp = SAMLSSOUtil.marshall(response);
+
+        return compressResponse(resp);
+
```

Comment:
Removed spaces in PR https://github.com/wso2/carbon-identity/pull/800/files#diff-99bf8d528cca774c8ad8936fee3ca73cR1238